### PR TITLE
[decoupled-execution] enable decoupled-execution for new genesis

### PR DESCRIPTION
**Pull request by zekun000**
## Motivation

enable decoupled-execution for new genesis

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/diem/diem/tree/main/developers.diem.com, and link to your PR here.)

## If targeting a release branch, please fill the below out as well

 * Justification and breaking nature (who does it affect? validators, full nodes, tooling, operators, AOS, etc.)
 * Comprehensive test results that demonstrate the fix working and not breaking existing workflows.
 * Why we must have it for V1 launch.
 * What workarounds and alternative we have if we do not push the PR.

### CODE REVIEW

File: testsuite/smoke-test/src/genesis.rs [hunk 1]
```diff
@@ -104,6 +105,9 @@ fn test_genesis_transaction_flow() {
         .wait_for_all_nodes_to_catchup(Instant::now() + Duration::from_secs(60))
         .unwrap();
 
+    println!("Wait for node to synchronize on same version.");
+    thread::sleep(Duration::from_secs(1));
```
Comment (yuxiamit) [inline testsuite/smoke-test/src/genesis.rs:109]: maybe add a `wait_for_same_version` like `wait_until_healthy`?

File: config/management/genesis/src/genesis.rs [hunk 2]
```diff
@@ -42,7 +42,12 @@ impl Genesis {
             .build(
                 chain_id,
                 None,
-                OnChainConsensusConfig::V1(ConsensusConfigV1 { two_chain: true }),
+                OnChainConsensusConfig::V2(ConsensusConfigV2 {
+                    two_chain: true,
+                    decoupled_execution: true,
+                    back_pressure_limit: 10,
+                    exclude_round: 20,
+                }),
```
Comment (gregnazario) [inline config/management/genesis/src/genesis.rs:50]: So, are these just hardcoded numbers that we don't want to configure with the tool?

File: testsuite/smoke-test/src/genesis.rs (hunk 1 above)
Comment (gregnazario) [reply testsuite/smoke-test/src/genesis.rs:109]: Oof time based tests :(

File: config/management/genesis/src/genesis.rs (hunk 2 above)
Comment (zekun000) [reply config/management/genesis/src/genesis.rs:50]: those number should work, we can configure it via txns as well

**Review by gregnazario — APPROVED** [review]

File: testsuite/forge/src/backend/local/swarm.rs [hunk 3]
```diff
@@ -132,6 +132,13 @@ impl LocalSwarmBuilder {
             SwarmDirectory::Temporary(TempDir::new()?)
         };
 
+        // Single node orders blocks too fast which would trigger backpressure and stall for 1 sec
+        // which cause flakiness in tests.
```
Comment (gregnazario) [inline testsuite/forge/src/backend/local/swarm.rs:136]: Lol, 2 fast 2 furious

**Review by gregnazario — APPROVED** [review]

File: testsuite/forge/src/backend/local/swarm.rs (hunk 3 above)
Comment (zekun000) [reply testsuite/forge/src/backend/local/swarm.rs:136]: hmm, seems CI is not happy about this slowdown for whatever reason, it works pretty well locally

**Review by gregnazario — DISMISSED** [review]

**Review by msmouse — APPROVED** [review]

**Pull request closed without merging**